ruby/app.rb

```diff
@@ -113,6 +114,65 @@
   end
 end
 
+# Create and then retrieve an Asset Report for one or more Items. Note that an
+# Asset Report can contain up to 100 items, but for simplicity we're only
+# including one Item here.
+# https://plaid.com/docs/#assets
+# rubocop:disable Metrics/BlockLength
+get '/assets' do
+  begin
+    asset_report_create_response =
+      client.asset_report.create([access_token], 10, {})
+    pretty_print_response(asset_report_create_response)
+  rescue Plaid::PlaidAPIError => e
+    error_response = format_error(e)
+    pretty_print_response(error_response)
+    content_type :json
+    error_response.to_json
+  end
+
+  asset_report_token = asset_report_create_response['asset_report_token']
+
+  asset_report_json = nil
+  num_retries_remaining = 1
```

Comment:
I think this should be 10 or 20 based on how it is set elsewhere -- when running this branch with only 1 retry I could never get the Ruby Quickstart to actually load an asset report successfully. We'd hit the `Timed out when polling for Asset Report` error which in turns causes a client-side JavaScript exception (specifically that `reportData.items` is undefined)